src/actions/shareArticleAction.js

```diff
@@ -0,0 +1,22 @@
+import { SHARE_ARTICLE } from "./types";
+
+export const ShareArticleFunction = (slug, mediaType) => dispatch => {
```

Comment:
Could you rename this action to a more descriptive name because this is misleading